### PR TITLE
Standardize instance number defaults

### PR DESCRIPTION
**Pull request by folksgl**
- Correct yamllint findings
- Set default number of instances to 2 for all services

### CODE REVIEW

File: sk-dbconfigs/manifest.yml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-  services:
-    - sk-postgres
-  env:
-    SK_SECRETS: ((SK_SECRETS))
+  - name: sk-dbconfigs
+    memory: 1M
+    processes:
+      - type: task
+        memory: 128M
+        instances: 1
```
Comment (shumatepf) [inline sk-dbconfigs/manifest.yml:9]: Should this be 2?

File: sk-esconfigs/manifest.yml [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-  services:
-    - sk-elasticsearch
-  env:
-    SK_SECRETS: ((SK_SECRETS))
+  - name: sk-esconfigs
+    memory: 1M
+    processes:
+      - type: task
+        memory: 128M
+        instances: 1
```
Comment (shumatepf) [inline sk-esconfigs/manifest.yml:9]: Should this be 2?

File: sk-manifest/manifest.yml [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
-  services:
-    - sk-postgres
-  env:
-    SK_SECRETS: ((SK_SECRETS))
+  - name: sk-manifest
+    memory: 1M
+    processes:
+      - type: task
+        memory: 128M
+        instances: 1
```
Comment (shumatepf) [inline sk-manifest/manifest.yml:9]: Should this be 2?

**Comment by folksgl** [comment]
@shumatepf see https://github.com/18F/identity-give-orchestration#tasks

The items you've referenced are not cloud foundry apps, but rather tasks that need to be run to initialize services. Once the task is complete, it terminates.

**Review by shumatepf — APPROVED** [review]

**Pull request merged**